src/components/staking/PageValidator.vue

```diff
@@ -595,7 +595,7 @@ span {
   line-height: normal;
 }
 
-.bech32-address {
+.copyable-address {
```

Comment:
why is this defined multiple times?

Reply:
/shrug — probably needs to be refactored